src/V3Width.cpp

```diff
@@ -3413,50 +3413,67 @@ class WidthVisitor final : public VNVisitor {
         // which member each AstPatMember corresponds to before we can
         // determine the dtypep for that PatMember's value, and then
         // width the initial value appropriately.
-        using PatMap = std::map<const AstMemberDType*, AstPatMember*>;
+        using PatMap = std::map<const AstMemberDType*, AstPatMember*>; // Store member: value
+        using DTypeMap = std::map<const std::string, AstPatMember*>; // Store data_type: default_value
```

Comment:
```suggestion
        using PatMap = std::map<const AstMemberDType*, AstPatMember*>;  // Store member: value
        using DTypeMap = std::map<const std::string, AstPatMember*>;  // Store data_type: default_value
```

Still not sure why the autoformatter didn't fix this for you...  I see the action did run.

Reply:
Sorry but when this auto formatter should work? Or in which part should this be checked?